rllib/utils/test_utils.py

```diff
@@ -317,13 +317,29 @@ def check_compute_single_action(
 
     action_space = pol.action_space
 
-    def _test(
-        what, method_to_test, obs_space, full_fetch, explore, timestep, unsquash, clip
+    policy_with_connectors_times = []
+    algo_times = []
+
+    def _test_compute_actions(
+        what, method_to_test, full_fetch, explore, timestep, unsquash, clip
     ):
         call_kwargs = {}
         if what is algorithm:
+            # Get the obs-space from Workers.env (not Policy) due to possible
```

Comment:
Moved this inside this test method because we need to raw obs space to test connectors